OpenSim/Common/ComponentSocket.h

```diff
@@ -807,6 +808,7 @@ class Input : public AbstractInput {
     const ChannelList& getChannels() const {
         return _connectees;
     }
+#endif
```

Comment:
Same question, as I'm not sure if ChannelList get mapped to an actual humanly usable class or not, if not then LGTM